internal/bminventory/inventory_test.go

```diff
@@ -14039,14 +13962,6 @@ var _ = Describe("GetInfraEnvDownloadURL", func() {
 			Expect(infraEnv.ExpiresAt.Equal(payload.ExpiresAt)).To(BeTrue())
 		})
 	})
-
-	It("when not using the image service it returns 400", func() {
```

Comment:
That's no longer an option? do we enforce it?

Reply:
Yes to both: https://github.com/openshift/assisted-service/blob/9f463c8ac54821c57089dd68a86a27fce43b4e5b/cmd/main.go#L204-L206